_requirements/perf.txt

```diff
@@ -1,3 +1,4 @@
 uvloop
 tenacity
 jsonargparse
+orjson==3.10.15
```

Comment:
requirements.txt contains the main dependency. These are optional.

Reply:
You asked to make this dependency optional in one of the comments